infoview/info.tsx

```diff
@@ -156,10 +157,16 @@ export function Info(props: InfoProps): JSX.Element {
         const goal = info.record && info.record.state;
         if (goal) copyToComment(goal);
     }
+    const matchFirstGoal = new RegExp(/^⊢ (([^\n]+\n)+[^\n]+)\n\n/m);
+    function insertFirstGoal() {
+        const goal = info.record && info.record.state;
```

Comment:
```suggestion
        const goal = info.record?.state;
```
I hope this already works in typescript.